mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -11,33 +11,34 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
 
 import json
 import typing
+import uuid
 from datetime import datetime, timezone
 
 import pandas as pd
 import sqlalchemy as db
```

Comment:
I suggest removing this alias, as it's somewhat confusing with the new `mlrun.model_monitoring.db` subpackage.
```suggestion
import sqlalchemy
```